library/Network/Wai/Cli.hs

```diff
@@ -13,19 +13,23 @@ import qualified Network.Wai.Handler.FastCGI as FCGI
 #endif
 import           Network.Wai.Middleware.RequestLogger (logStdoutDev)
 import           Network.HTTP.Types (serviceUnavailable503)
+#ifdef UnixSockets
```

Comment:
Other options are called `WaiCliSomething`, this one should be too

Reply:
`WaiCliUnix` it is...